README.md

```diff
@@ -22,6 +22,7 @@ COMMANDS:
 GLOBAL OPTIONS:
    --scope value  Scope of command. For example: event, build, pipeline
    --type value   Type of the command. For example: cache, artifacts, steps (default: "stable")
+   --timeout value   Specifies the timeout in seconds. (default: 60(get) or 300(set,remove))
```

Comment:
I think the description would be better if it were something like this.
```
Timeout of command in seconds. A Timeout of zero means no timeout.(default: 60(get), 300(set, remove))
```

Reply:
Thank you for your comments. I think that's a good point.
It would be helpful if we included the option for zero.
However, I don't think this option should be used by end-users.
I consider the zero option to be more for operators, so I think it's not necessary to include it in the help section.